resources/startup.sh

```diff
@@ -28,17 +30,35 @@ create_truststore.sh /opt/scm-server/conf/truststore.jks > /dev/null
 # create ca certificate store for mercurial
 create-ca-certificates.sh /opt/scm-server/conf/ca-certificates.crt
 
-if ! [ -d "/var/lib/scm/config" ];  then
-	mkdir -p "/var/lib/scm/config"
+if ! [ -d "${SCM_DATA}/config" ];  then
+	mkdir -p "${SCM_DATA}/config"
 fi
 
-# install plugins
-if ! [ -d "/var/lib/scm/plugins" ];  then
-	mkdir -p "/var/lib/scm/plugins"
-
-  /usr/local/bin/scm-plugin-snapshot -config /etc/scm/plugin-config.yml /var/lib/scm/plugins
+# delete outdated plugins
+if [ -a "${SCM_DATA}/plugins/delete_on_update" ];  then
+  rm -rf "${SCM_DATA}/plugins"
 fi
 
+start_scm_server () {
```

Comment:
Why not keep the term "core plugins" or maybe "dogu core plugins" since this plugins are not deletable and critical for the scm dogu. Would be nice to tell apart this two plugins from the inital set of plugins defined in the install-plugins.groovy script.